### DISCUSSION TITLE
[678900] 农夫王朝 Farmer's Dynasty（本体）

### DISCUSSION THREAD

**Discussion by Dew778** [pjy612/SteamManifestCache#381] [Share]
`
appid：678900
`

```
678901;7db9128b587a8d1c55382f39d671eb66234504ad5403e2fc8cebb05b15e959f9
```

[678900_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13972731/678900_config.vdf.txt)
[[678900] 农夫王朝 Farmer's Dynasty（本体）清单.zip](https://github.com/pjy612/SteamManifestCache/files/13973059/678900.Farmer.s.Dynasty.zip)

Code：7337BCEA697F2EBE
Email:3394748293@qq.com

**Comment by ZEERDEER** [comment]
标题格式 #26

**Reply by Dew778** [reply]
已修改
